.eslintrc.js

```diff
@@ -223,5 +224,18 @@ module.exports = {
 				],
 			},
 		],
+		// Disabled for now until we finish the migration
+		'wpcalypso/no-package-relative-imports': [
+			'off',
+			{
+				mapping: [
+					{
+						dir: path.join( __dirname, 'client' ),
```

Comment:
This means: For every import like `import * from 'X'`, when X is a subdirectory of the path specified in `dir`, replace it with `wp-calypso/X`